lazyflow/operators/ioOperators/opStreamingHdf5Reader.py

```diff
@@ -86,10 +86,16 @@ def setupOutputs(self):
         assert len(axistags) == len( dataset.shape ),\
             "Mismatch between shape {} and axisorder {}".format( dataset.shape, axisorder )
 
-        # Configure our slot meta-info
         self.OutputImage.meta.dtype = dataset.dtype.type
-        self.OutputImage.meta.shape = dataset.shape
-        self.OutputImage.meta.axistags = axistags
+        if 'c' not in axisorder:
+            self.OutputImage.meta.shape = (*dataset.shape, 1)
+            axisorder = get_default_axisordering(self.OutputImage.meta.shape)
+            axistags = vigra.defaultAxistags(axisorder)
+            self.OutputImage.meta.axistags = axistags
+        else:
+            # Configure our slot meta-info
```

Comment:
this comment should come before the `if 'c' not in axisorder:` line, as in both branches, the meta-info is configured